sql/2022/page-weight/compression-usage.sql

```diff
@@ -0,0 +1,14 @@
+SELECT
+  _TABLE_SUFFIX AS client,
+  COUNTIF(JSON_VALUE(report, '$.audits.text-compression.score') IS NULL) AS null_count,
+  COUNTIF(SAFE_CAST(JSON_VALUE(report, '$.audits.text-compression.score') AS FLOAT64) >= 0.9) AS pass_count,
+  COUNTIF(SAFE_CAST(JSON_VALUE(report, '$.audits.text-compression.score') AS FLOAT64) < 0.9) AS fail_count
```

Comment:
For each of these "count" queries, we should also calculate the percent of pages. Something like this:
```suggestion
  COUNTIF(SAFE_CAST(JSON_VALUE(report, '$.audits.text-compression.score') AS FLOAT64) >= 0.9) AS pass_count,
  COUNTIF(SAFE_CAST(JSON_VALUE(report, '$.audits.text-compression.score') AS FLOAT64) < 0.9) AS fail_count,
  COUNT(0) AS total,
  COUNTIF(SAFE_CAST(JSON_VALUE(report, '$.audits.text-compression.score') AS FLOAT64) >= 0.9) / COUNT(0) AS pct_pass,
  COUNTIF(SAFE_CAST(JSON_VALUE(report, '$.audits.text-compression.score') AS FLOAT64) < 0.9) / COUNT(0) AS pct_fail
```

Can you also update the rest of the queries to calculate percents?

Reply:
@drohe let me know if I can help at all with this. Want to make sure we give @fellowhuman1101 enough time to write the draft.